stripe-patrons-data/src/main/scala/com/gu/patrons/lambdas/PatronCancelledEventLambda.scala

```diff
@@ -40,9 +40,13 @@ import scala.concurrent.duration.{Duration, DurationInt, MINUTES}
 import scala.jdk.CollectionConverters.MapHasAsScala
 import scala.util.{Failure, Success, Try}
 
-object PatronCancelledEventLambda extends StrictLogging {
+class PatronCancelledEventLambda extends StrictLogging {
   val runner = configurableFutureRunner(60.seconds)
 
+  implicit val stage = StageConstructors.fromEnvironment
+  private val stripeConfig = PatronsStripeConfig.fromParameterStoreSync(stage)
+  private val identityConfig = PatronsIdentityConfig.fromParameterStoreSync(stage)
```

Comment:
by making these Sync, it means they will run one after the other instead of in parallel.  I'm not sure how much benefit of running them in parallel, but if there is we could always sequence and then resolve the Futures into a val using Await or similar.

On the flip side, if only warming up AWS client is important, and subsequent fetches are cheap, we could just fetch one of the config and throw it away, and then refetch it all once the lambda is woken up again.  This would make things easier if we update the config, as we wouldn't have to redeploy/restart the lambda to pick it up.

Reply:
>> we could just fetch one of the config and throw it away
I hadn't considered this! It's a bit hacky :)
I think I'm coming round to the idea of tying a snapshot of the config to a deploy. In a way it's closer to how ec2 apps work